defs/scenarios/openstack/neutron/ovn_stale_db.yaml

```diff
@@ -0,0 +1,58 @@
+vars:
+  msg_common: >-
+    is reporting frequent reconnections to the ovn
+    southbound database due to "stale data" and this may require taking
+    action to resolve. One cause is if you have recently rebuilt your
+    southbound database. See the linked bug for more information. One
+    suggested workaround is to
+checks:
+  is_neutron_server:
+    apt: neutron-server
+  is_neutron_ovn_metadata_agent:
+    apt: neutron-ovn-metadata-agent
+  has_1829109_neutron_log:
+    input:
+      - var/log/neutron/neutron-server.log
+      - var/log/neutron/neutron-ovn-netadata-agent.log
```

Comment:
Typo: metadata - i'm assuming it's using that path to find the file

Reply:
Ha well spotted. I'll get that fixed.